native-engine/datafusion-ext-commons/src/spark_hash.rs

```diff
@@ -51,7 +51,7 @@ pub fn create_hashes<T: num::PrimInt>(
     if arrays.is_empty() {
         return vec![seed; len];
     }
-    let mut hash_buffer = vec![T::zero(); len];
+    let mut hash_buffer = vec![seed; len];
```

Comment:
i think this should be fixed in https://github.com/kwai/blaze/blob/6d327db32d0ab39dbad2b6d679145e4167537f2c/native-engine/datafusion-ext-commons/src/spark_hash.rs#L153, otherwise it breaks compatibility to spark's shuffle hash behavior.